update usability test changes post to reflect design updates (#194)
DFE-Digital/teacher-cpd-design-history: app/posts/register-for-an-npq/2023-05-30-usability-test-changes.md

Users do not always understand the purpose of creating a DfE Identity account, or that they can log back in. 

In usability sessions, we saw:

> 1. Users skim through this page quickly or don't read it.

Reviewer: ```suggestion
1. skimmed through the page quickly or did not read it```
